include/tvm/topi/transform.h

```diff
@@ -23,6 +23,9 @@
  */
 #ifndef TVM_TOPI_TRANSFORM_H_
 #define TVM_TOPI_TRANSFORM_H_
+#define NPY_MAXDIMS 16
+#define NPY_MAXARGS 16
+#define MAXAXIS 128
```

Comment:
These constants seems to be not necessary

Reply:
Removed. thx